samples/net/echo_server/src/echo-server.c

```diff
@@ -144,3 +141,61 @@ void main(void)
 		stop_udp();
 	}
 }
+
+static void do_service(k_work_handler_t handler)
+{
+	static struct k_work work;
+
+	k_work_init(&work, handler);
+
+	k_work_submit(&work);
+}
+
+static void event_handler(struct net_mgmt_event_callback *cb,
+			  u32_t mgmt_event, struct net_if *iface)
+{
+	if (mgmt_event == NET_EVENT_L4_READY) {
+		NET_INFO("Network connectivity ready.");
+
+		do_service(start_udp_and_tcp);
+
```

Comment:
This will call do_service(start_udp_and_tcp) once for each IP version configured for the interface, right? Is this the intention behind the echo_server logic? Shouldn't echo_server also know which of the N interfaces is going to be ready now? What actually happens when echo server gets more than one event like this?